acbuild/cat-manifest.go

```diff
@@ -34,6 +33,7 @@ var (
 func init() {
 	cmdAcbuild.AddCommand(cmdCat)
 	cmdCat.Flags().BoolVar(&prettyPrint, "pretty-print", false, "Print the manifest with whitespace")
+	cmdCat.Flags().BoolVar(&printConfig, "config", false, "(oci only) Prints the image config instead of the manifest")
```

Comment:
What about turning this into a `--type` or `--mode` enum? Boolean flags are harder to accomodate/extend for future cases.